AutolayoutPractice/RemindersList/Views/RemindersListVC.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  RemindersListVC.swift
+//  AutolayoutPractice
+//
+//  Created by Amanda Benavente on 6/17/24.
+//
+
+import Foundation
+import UIKit
+
+class RemindersListVC: UIViewController {
+    let tableView = UITableView()
+    let viewModel = RemindersViewModel()
+    
+    override func viewDidLoad() {
+        self.title = "Reminders"
+        configureTableView()
+    }
+    
+    func configureTableView() {
+        view.addSubview(tableView)
+        tableView.frame = view.bounds
+        tableView.estimatedRowHeight = 90
+        tableView.rowHeight = UITableView.automaticDimension
+        tableView.delegate = self
+        tableView.dataSource = self
+        tableView.register(ReminderCell.self, forCellReuseIdentifier: ReminderCell.identifier)
+    }
+}
+
+extension RemindersListVC: UITableViewDelegate, UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        viewModel.reminders.count
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: ReminderCell.identifier, for: indexPath) as? ReminderCell else {
+            return UITableViewCell()
+        }
+        let reminder = viewModel.reminders[indexPath.row]
+        cell.reminder = reminder
```

Comment:
`cell.reminder = ReminderCellConfiguration(title: reminder.title, subtitle: reminder.subt....)`

Reply:
Once our application is complex enough to warrant the need for a builder:

`cell.reminder = reminderFeatureUIBuilder.buildReminderUIConfig(reminder: reminder)`